packages/eslint-plugin-svelte/tests/src/configs/all.ts

```diff
@@ -1,16 +1,20 @@
 import assert from 'assert';
-import semver from 'semver';
 import plugin from '../../../src/index.js';
-import { ESLint } from '../../utils/eslint-compat.js';
+import { loadESLint, type ESLint as ESLintClass } from 'eslint';
```

Comment:
as far as i understand, `ESLint` here is the flat ESLint already. but when we force install 8.x in CI, it will be the legacy one at runtime. which is why we can't just `import {ESLint} from 'eslint'`